### ISSUE TITLE
Using a customer scorer that takes y_prob instead of y_pred? (brier_score_loss)

### ISSUE THREAD

**Issue by soliverc** [EpistasisLab/tpot#968]
I am trying to use brier_score_loss as a custom scorer, but it takes `y_prob` rather than `y_pred`: https://scikit-learn.org/stable/modules/generated/sklearn.metrics.brier_score_loss.html 

Is there any way to implement this? 

```python
from sklearn.metrics import brier_score_loss

tpot = TPOTClassifier(generations=1, 
                      population_size=100,
                      verbosity=2,
                      n_jobs=-2,
                      scoring=brier_score_loss,
                      template='Classifier')
```
This is the error when I try to use it:

```python
C:\Users\seanc\Anaconda3\lib\site-packages\tpot\base.py:350: DeprecationWarning: Scoring function <function brier_score_loss at 0x000001A1314D2D08> looks like it is a metric function rather than a scikit-learn scorer. This scoring type was deprecated in version TPOT 0.9.1 and will be removed in version 0.11. Please update your custom scoring function.
  'Please update your custom scoring function.'.format(scoring), DeprecationWarning)
```

**Comment by weixuanfu** [comment]
You could just try to scoring="brier_score_loss" since it is available in scikit-learn `SCORERS` object (see this [link](https://scikit-learn.org/stable/modules/model_evaluation.html#common-cases-predefined-values)). You may need update scikit-learn for using it. 


Or if you have have a customer scorer with y_prob.  try to use [`make_scorer`](https://scikit-learn.org/stable/modules/generated/sklearn.metrics.make_scorer.html) function for creating a scorer for scoring parameter in `TPOTClassifier`. Please check [the link](https://stackoverflow.com/questions/29664657/creating-scorer-for-brier-score-loss-in-scikit-learn) or [this one](https://github.com/scikit-learn/scikit-learn/blob/1495f69242646d239d89a5713982946b8ffcf9d9/sklearn/metrics/scorer.py#L502) for examples.
